manifests/init.pp

```diff
@@ -203,6 +203,9 @@
 #
 # $proxy_password::             Proxy password for authentication
 #
+# $yum_max_speed::              Maximum bandwidth used per download in kb/sec per worker
```

Comment:
@sean797 @jlsherrill I changed this to KB since we are using the to_bytes that Sean suggested. Will that work for the description?